apps/src/sites/studio/pages/lessons/edit.js

```diff
@@ -83,7 +85,7 @@ $(document).ready(function() {
   const store = getStore();
 
   //TODO Switch to using real data once we have activity data
-  store.dispatch(init(activities, levelKeyList));
+  store.dispatch(init(activities, levelKeyList, searchOptions));
```

Comment:
I haven't totally wrapped my head around this, but should levelKeyList start as an empty array now instead of the sample data?

Reply:
It looks like the only place we are using levelKeyList is to get the name of the level in the RemoveLevelDialog. I'm gonna create a PR to remove levelKeyList.

Reply:
Nevermind my search wasn't showing up fully. There was more it impacts. I can look into this.